### ISSUE TITLE
should not disable bosh agent on bootstrap & inception vms

### ISSUE THREAD

**Issue by poblin-orange** [orange-cloudfoundry/paas-templates#511]
### Observed behavior
As of v41, the inception and bootstrap vm have their bosh-agent disabled. This configuration seems herited from bbl jumpbox, but causes issues in our use case. We need a persistent disk, and disabling the bosh-agent prevents vm to remount the disk on restart, and have bosh jobs started

(Issues seen on vsphere v40 env)

### Affected release

Reproduced on version 40, 41, 42

**Comment by gberche-orange** [comment]
Upstream issue reported at https://github.com/cloudfoundry/jumpbox-deployment/issues/28

Tested that removing this job in bbl solves the symptoms observed

**Comment by gberche-orange** [comment]
Partial workaround: renable the bosh agent, and wait for the bosh agent to mount to the persistent disks. Not clear whether this fixes monit and bosh jobs

```bash
cd /var/vcap/bosh/bin/
ln -s ./bosh-agent-os-conf-disabled bosh-agent
```

**Comment by poblin-orange** [comment]
fixed in v42.0.4
